qutebrowser/config/configtypes.py

```diff
@@ -1160,31 +1181,21 @@ def validate(self, value):
         value = os.path.expandvars(value)
         value = os.path.expanduser(value)
         try:
-            if not os.path.isabs(value):
-                # probably a CSS, so we don't handle it as filename.
-                # FIXME We just try if it is encodable, maybe we should
-                # validate CSS?
-                # https://github.com/The-Compiler/qutebrowser/issues/115
-                try:
-                    value.encode('utf-8')
-                except UnicodeEncodeError as e:
-                    raise configexc.ValidationError(value, str(e))
-                return
-            elif not os.path.isfile(value):
-                raise configexc.ValidationError(value, "must be a valid file!")
-        except UnicodeEncodeError as e:
-            raise configexc.ValidationError(value, e)
-
-    def transform(self, value):
-        path = os.path.expandvars(value)
-        path = os.path.expanduser(path)
-        if not value:
-            return None
-        elif os.path.isabs(path):
-            return QUrl.fromLocalFile(path)
-        else:
-            data = base64.b64encode(value.encode('utf-8')).decode('ascii')
-            return QUrl("data:text/css;charset=utf-8;base64,{}".format(data))
+            super().validate(value)
+        except configexc.ValidationError:
+            try:
+                if not os.path.isabs(value):
+                    # probably a CSS, so we don't handle it as filename.
+                    # FIXME We just try if it is encodable, maybe we should
+                    # validate CSS?
+                    # https://github.com/The-Compiler/qutebrowser/issues/115
+                    try:
+                        value.encode('utf-8')
+                    except UnicodeEncodeError as e:
+                        raise configexc.ValidationError(value, str(e))
+                    return
```

Comment:
I didn't see this before (in my code), but this should be simplified to a simple `value.encode('utf-8')` because `UnicodeEncodeError` is handled below already anyways, and it's the end of the function anyways.